Modules/Loadable/Markups/MRML/vtkCurveMeasurementsCalculator.h

```diff
@@ -96,12 +115,16 @@ class VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkCurveMeasurementsCalculator : pub
   /// Flag determining whether the filter should calculate curvature
   bool CalculateCurvature{false};
 
+  /// Flag determining whether the filter should calculate torsion
+  bool CalculateTorsion{false};
+
   /// Command handling data array modified events
   vtkCallbackCommand* ControlPointArrayModifiedCallbackCommand;
   /// List of observed control point arrays (for removal of observations)
   vtkCollection* ObservedControlPointArrays;
 
-  std::string CurvatureUnits;
+  std::string CurvatureUnits{"mm-1"};
```

Comment:
It would be nice to set this based on the length unit in the scene.

Reply:
See `vtkMRMLMarkupsCurveNode::UpdateMeasurementsInternal`. I added torsion unit setting now.